class-sc-wp-updates-notifier.php

```diff
@@ -743,6 +875,60 @@ public function sc_wpun_settings_main_field_hide_updates() {
 			</select>
 			<?php
 		}
+
+		/** Email settings **/
+		public function sc_wpun_settings_email_notifications_text() {
```

Comment:
Empty function, can you remove?

Reply:
this can not be removed, its required for the settings section